pkg/config/config.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+package config
+
+import (
+	"os"
+	"path/filepath"
+	"strconv"
+
+	"github.com/golang/glog"
+)
+
+const (
+	DEFAULT_SERVICE_PORT = ":3030"
+	DEFAULT_HTTP_TIMEOUT = 300000                                  // 5 min, to fix the EOF response at the collector
+	DEFAULT_USE_MOCK     = false                                   // Use Mocked Cluster ID ?
+	DEFAULT_CCX_SERVER   = "http://localhost:8080/api/v1/clusters" // For local use only
+)
+
+// Config - Define a config type to hold our config properties.
+type Config struct {
+	ServicePort string
+	HTTPTimeout int  // timeout when the http server should drop connections
+	UseMock     bool // Use Mock Server or actual endpoint
+	CCXServer   string
+	KubeConfig  string `env:"KUBECONFIG"` // Local kubeconfig path
```

Comment:
All these could be read from environment variables.
```suggestion
	ServicePort string `env:"SERVICE_PORT"`
	HTTPTimeout int `env:"HTTP_TIMEOUT"` // timeout when the http server should drop connections
	UseMock     bool `env:"USE_MOCK"` // Use Mock Server or actual endpoint
	CCXServer   string `env:"CCX_SERVER"`
	KubeConfig  string `env:"KUBECONFIG"` // Local kubeconfig path
```

Reply:
Done